test/schema/home/home_page_artwork_modules.js

```diff
@@ -109,6 +116,7 @@ describe('HomePageArtworkModules', () => {
           'followed_galleries',
           'saved_works',
           'current_fairs',
+          'followed_artist',
           'related_artists',
```

Comment:
Not sure they are logically connected (see Brian's comments on #490). I'm fine to do either (make a hard group, or allow them to be re-ordered) way. I'd imagine grouping them together would take a bit more work to sync (with both force and emission).